app/rax/package.json

```diff
@@ -47,7 +47,6 @@
   },
   "peerDependencies": {
     "@babel/core": "*",
-    "babel-loader": "^7.0.0 || ^8.0.0",
```

Comment:
Nice! this means we can remove some documentation too, I think we tell users to install this?